docs/security/trust/trust_sandbox.md

```diff
@@ -61,146 +63,114 @@ work exclusively with the Docker Hub, you would not need with these components.
-        Step 1 : FROM debian:jessie
-         ...
-         Successfully built 5683f17e9d72
-
+        Step 1 : FROM docker:dind
+        ...
+        Successfully built 34efc7e5f153
 
 ### Build and start up the trust servers
 
 In this step, you get the source code for your notary and registry services.
 Then, you'll use Docker Compose to build and start them on your local system.
 
-1. Change to back to the root of your  `notarysandbox` directory.
-
-        $ cd notarysandbox
-
-2. Clone the `notary` project.
+1. Clone the `notary` project.
 
-          $ git clone -b trust-sandbox https://github.com/docker/notary.git
+          $ git clone https://github.com/docker/notary.git
 
-3. Clone the `distribution` project.
-
-        $ git clone https://github.com/docker/distribution.git
-
-4. Change to the `notary` project directory.
+2. Change to the `notary` project directory.
 
         $ cd notary
 
    The directory contains a `docker-compose` file that you'll use to run a
    notary server together with a notary signer and the corresponding MySQL
    databases. The databases store the trust information for an image.
 
-5. Build the server images.
+3. Build the server images.
 
         $  docker-compose build
 
     The first time you run this, the build takes some time.
 
-6. Run the server containers on your local system.
+4. Run the server containers on your local system.
 
         $ docker-compose up -d
 
     Once the trust services are up, you'll setup a local version of the Docker
     Registry v2.
 
-7. Change to the `notarysandbox/distribution` directory.
-
-8. Build the `sandboxregistry` server.
-
-        $ docker build -t sandboxregistry .
+5. Run the `sandboxregistry` server based on the the public registry image.
 
-9. Start the `sandboxregistry` server running.
+        $ docker run -d -p 5000:5000 --net notary_sig --name sandboxregistry registry:2.4.1
 
-        $ docker run -p 5000:5000 --name sandboxregistry sandboxregistry &
+    The first time you run this, it will download the registry image from Docker Hub
+    before running it.
 
 ## Playing in the sandbox
 
-Now that everything is setup, you can go into your `notarysandbox` container and
+Now that everything is setup, you can go into your `trustsandbox` container and
 start testing Docker content trust.
 
 
-### Start the notarysandbox container
+### Start the trustsandbox container
 
-In this procedure, you start the `notarysandbox` and link it to the running
-`notary_notaryserver_1` and `sandboxregistry` containers. The links allow
+In this procedure, you start the `trustsandbox` container and link it to the running
+`notary_server_1` and `sandboxregistry` containers. The links allow
 communication among the containers.
 
 ```
-$ docker run -it -v /var/run/docker.sock:/var/run/docker.sock --link notary_notaryserver_1:notaryserver --link sandboxregistry:sandboxregistry notarysandbox
-root@0710762bb59a:/#
+$ docker run -d --privileged --net notary_sig \
```

Comment:
Yes, thinking what would make sense; I see the docker-compose file in the notary repository also builds notary from source; if that's something we want, it'd best be somewhere in the notary repository.

Perhaps having images available on Docker Hub would be good (I see no direct reason why people would have to build from source; especially since we don't tell them to checkout a specific tag/branch, so it's possible current master doesn't match this guide